proposals/nnnn-AOA-multiplexing.md

```diff
@@ -9,45 +9,64 @@
  
 ## Proposed solution
 
-Implement multiplexing functionality to AOA transport of SDL Android library. This architecture is similar to existing BT/SPP multiplexing. However, in Android, SPP and AOA may exist in parallel. So, another multiplexing router for AOA should be added. If OEM using other protocol on AOA for none SDL service to be supported then, this architecture can be used for this purpose with customized SDL Android library which is special router service for other standard SDL apps.
-SDL core is unnecessary to be changed. SDL core already supported multi session in single transport with using session id.
-Optionally, if HU manufacture to support multiplexing AOA only then AOA parameter can be "SDL", "Core", __"1.1"__, (not "SDL", "Core", __"1.0"__). 
-The spec must be written in Guide->Android->Getting Started->Using AOA protocol.
+Implement multiplexing functionality to AOA transport of SDL Android library. This architecture is similar to existing BT/SPP multiplexing. However, in Android, SPP and AOA may exist in parallel. So, another multiplexing router service for AOA will be added. SDL core is unnecessary to be changed because SDL core already supported multi session in single transport with using session id.
+If HU manufacture supports only multiplexing AOA app then, AOA parameter must be set as "SDL", "Core", __"1.1"__. On the other hands, if HU manufacture supports both multiplexing and single AOA app then, AOA parameter can be set as "SDL", "Core", __"1.0"__.
+App which is linked SDL proxy of multiplexing AOA support version, must include accessory_filter.xml as followings:
+```
+<usb-accessory manufacturer="SDL" model="Core" version="1.0"/>
+<usb-accessory manufacturer="SDL" model="Core" version="1.1"/> <!-- AOA multiplexing support -->
+```
+The spec must be applied to "SDLC web"->Guide->Android->Getting Started->Using AOA protocol.
+
+Optionally, even if some of OEMs/Tier1s are already using AOA for not SDL then, this proposal architecture can be applied with customized SDL Android library which includes customized router service to arbitrate SDL session and proprietary service using AOA.  
+ex. proprietary AOA parameter (in customized router service) App + standard AOA parameter Apps.
+* Special HU (with proprietary AOA parameter) <-> customized router service <-arbitrate to SDL or proprietary service-> standard SDL apps or proprietary service
+* Standard SDL HU (with standard SDL AOA parameter) <-> one of standard SDL app <-> other standard SDL apps.
+This is also same architecture to BT/SPP multiplexing.
 
 ## Potential downsides
 
-Downside:
-backward compatibility
-App vendor may worry AOA parameter setting and proxy version (don't worry to set both, new AOA parameter and old AOA parameter. old HU can connect multi AOA apps which are linked new sdl proxy).
+Backward compatibility:  
+There is no change in SDL core. So that, new App linked SDL proxy which is modified as this proposal, can connect to old HU, too.
```

Comment:
Can you please reword the second sentence?

Reply:
@theresalech 
"So that, new App with new AOA-mutilplexing-SDL-proxy can connect to old HU, too.",  this is revised one. Is this clarified?

Reply:
Yes, I think this is okay.  Thanks!

Reply:
Thanks. I'll commit with other comment.